xray/src/bin/build_xray_quadtree.rs

```diff
@@ -61,6 +61,15 @@ fn parse_arguments() -> clap::ArgMatches<'static> {
                 .long("min_intensity")
                 .takes_value(true)
                 .required_if("coloring_strategy", "colored_with_intensity"),
+            clap::Arg::with_name("max_stddev")
+                .help(
+                    "Maximum stddev for colored_with_height_stddev. Every stddev above this \
```

Comment:
Nit: Maximum standard deviation for colored_with_height_stddev?

standard deviation as stddev isn't very obvious to me. But, I could be wrong.